src/site/content/en/blog/module-workers/index.md

```diff
@@ -0,0 +1,232 @@
+
+##### worker.js:
+
+```js
+import { sayHello } from './greet.js';
+addEventListener('message', e => {
+  postMessage(sayHello());
+});
+```
+
+##### greet.js:
+
+```js
+import greetings from './data.js';
+export function sayHello() {
+  return greetings.hello;
+}
+```
+
+To ensure great performance, the old `importScripts()` method is not available within module
+workers. Switching workers to use JavaScript modules means all code is loaded in [strict
+mode](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode). Another
+notable change is that the value of `this` in the top-level scope of a JavaScript module is
+`undefined`, whereas in classic workers the value is the worker's global scope. Fortunately, there
+has always been a `self` global that provides a reference to the global scope. It's available in
+all types of workers including service workers, as well as in the DOM.
+
+{% Aside %}
+Module workers also remove support for HTML-style comments.
+{% endAside %}
+
+## Preload workers with modulepreload
+
+One substantial performance improvement that comes with module workers is the ability to preload
+workers and their dependencies. With module workers, scripts are loaded and executed as standard
+JavaScript modules, which means they can be preloaded and even pre-parsed using `modulepreload`:
+
+```html
+<!-- preloads worker.js and its dependencies: -->
+<link rel="modulepreload" href="worker.js">
+
+<script>
+  addEventListener('load', () => {
+    // our worker code is likely already parsed and ready to execute!
+    const worker = new Worker('worker.js', { type: 'module' });
+  });
+</script>
+```
+
+Preloaded modules can also be used by both the main thread and module workers. This is useful for
+modules that are imported in both contexts, or in cases where it's not possible to know in advance
+whether a module will be used on the main thread or in a worker.
+
+Previously, the options available for preloading web worker scripts were limited and not
+necessarily reliable. Classic workers had their own "worker" resource type for preloading, but no
+browsers implemented `<link rel="preload" as="worker">`. As a result, the primary technique
+available for preloading web workers was to use `<link rel="prefetch">`, which relied entirely
+on the HTTP cache. When used in combination with the correct caching headers, this made it possible
+to avoid worker instantiation having to wait to download the worker script. However, unlike
+`modulepreload` this technique did not support preloading dependencies or pre-parsing.
+
+<div class="glitch-embed-wrap" style="height: 480px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/worker-preloading?previewSize=100&attributionHidden=true"
+    alt="worker-preloading on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+## What about Service Worker?
```

Comment:
```suggestion
## What about service worker?
```

Nit: I can't find our own personal guidance on this, but service worker should be lower case. The spec backs this convention up: https://www.w3.org/TR/service-workers/#dfn-service-worker